DSL/Ruuter.private/POST/rasa/slots/slotById.yml

```diff
@@ -1,6 +1,6 @@
 assign_values:
   assign:
-    parameters: ${incoming.body}
+    _slot: ${incoming.body.slot}
```

Comment:
I think is better to name the variable without an underscore at the start for better readability, could you please rename it

Reply:
Done